components/label.json

```diff
@@ -2,35 +2,22 @@
   "label": {
     "comment": "Applied to non editable pieces of text.",
     "primary": {
-      "#normal": {
-        "text": "@theme-text-primary-normal"
-      },
-      "#disabled": {
```

Comment:
why do you take out the disabled state from this text?

Reply:
As written in the above comment, I can't use it as it is defined here. I need the `disabled` state as an independent state, just like `primary` or `secondary` are defined.

Reply:
Pushed commit
[Reverted label changes](https://github.com/momentum-design/tokens/pull/212/commits/68e0bce65663e5a02fff04da65163dee7b51e904)
Changing this conversation to Resolved as it no longer applies.